lib/cambiatus_web/views/rich_link_view.ex

```diff
@@ -0,0 +1,16 @@
+defmodule CambiatusWeb.RichLinkView do
+  use CambiatusWeb, :view
+
+  require Earmark
+  require HtmlSanitizeEx
+
+  def md_to_txt(markdown) do
+    with {:ok, string, _} <- Earmark.as_html(markdown) do
+      HtmlSanitizeEx.strip_tags(string)
+      |> String.trim()
```

Comment:
if there is a pipe, use it all the way:

```suggestion
      string
      |> HtmlSanitizeEx.strip_tags()
      |> String.trim()
```